client/static/js/lib/fetch.js

```diff
@@ -0,0 +1,30 @@
+async function getPostData(id) {
+	try {
+		const url = `http://localhost:5000/posts/${id}`;
+		const response = await fetch(url);
+		const postData = await response.json();
+		return postData;
+	} catch (err) {
+		console.error(err);
+	}
+}
+
```

Comment:
Maybe add a route to show all posts in db? Not sure its completely necessary just finding something to criticize lol

Reply:
The client didn't pay for that feature as is not included in the decription of the job D: